content/client-server-api/modules/voip_events.md

```diff
@@ -171,18 +171,34 @@ In response to an incoming invite, a client may do one of several things:
 
 ##### Streams
```

Comment:
@richvdh [said](https://github.com/matrix-org/matrix-spec/pull/1602/files#r1287259846):

> can we add something to https://pr1602--matrix-spec-previews.netlify.app/client-server-api/#common-fields about the new field?

Reply:
I'm not sure about it, it only applies to 3 out of 7 event types, contrary to the other fields.

Reply:
Mmmhmm. Makes sense.